components/ButtonLike/index.js

```diff
@@ -0,0 +1,62 @@
+import { useRouter } from "next/router";
+import Button from "../Button";
+import SVGIcon from "../SVGIcon";
+
+export default function ButtonLike({ post, loggedInUserID, reload }) {
+  async function like() {
+    try {
+      const response = await fetch(`/api/posts/${post._id}`, {
+        method: "PUT",
+        body: JSON.stringify(loggedInUserID),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+
+      if (response.ok) {
+        const data = await response.json();
+      } else {
+        console.error(`Error: ${response.status}`);
+      }
```

Comment:
Hi Niklas,
do you need "data" here? It looks like you are not doing anything with it.
Maybe you could use something like that?
`if (!response.ok) {
console.error(`Error: ${response.status}`)
}`

Reply:
True, thank you :)